tests/test_utils_google_directions.py

```diff
@@ -455,19 +561,34 @@ def test_sending_requests(mocker, google_directions_api_response):
 def test_sending_requests_throws_error_if_key_not_found(mocker):
     mocker.patch.object(secrets_vault, 'get_google_directions_api_key', return_value=None)
     with pytest.raises(RuntimeError) as e:
-        google_directions.send_requests({})
+        google_directions.send_requests({}, None)
     assert 'API key was not found' in str(e.value)
 
 
 def test_parsing_routes_with_a_good_response(google_directions_api_response, generated_request):
+    legs = google_directions_api_response.json()['routes'][0]['legs']
+    expected_speed = sum([leg['distance']['value'] for leg in legs]) / sum([leg['duration_in_traffic']['value'] for leg in legs])
+
     data = google_directions.parse_routes(google_directions_api_response, generated_request['path_polyline'])
-    assert_semantically_equal(data, {'google_speed': 3.7183098591549295, 'google_polyline': 'ahmyHzvYkCvCuCdDcBrB'})
+    assert_semantically_equal(data, {'google_speed': expected_speed, 'google_polyline': 'ahmyHzvYkCvCuCdDcBrB'})
```

Comment:
It would be clearer to assert against the polyline in the fixture, rather than against a hardcoded value. You could also then change the value in the fixture and the test wouldn't break.